### DISCUSSION TITLE
Welcome to aptly Discussions!

### DISCUSSION THREAD

**Discussion by neolynx** [aptly-dev/aptly#1309] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by GreatEarl** [comment]
Hi !
We've been using aptly for a some time and it's quiet handy atm. We use it to deploy software updates OTA to our devices via AWS IoT core.

Is there any way to make aptly pull packages from other apt repos/resources?
Let's say it pulls some packages from 3-rd party repo and adds them into snapshot and publishes. 
This is due to the final consumer is able to pull from aptly atm.
Cheers.
